src/content/docs/style-guide/writing-docs/writer-workflow/tech-writer-workflow.mdx

```diff
@@ -55,72 +55,81 @@ Need to edit a doc? Use this table to decide where to work!
 **Continue reading for instructions on how to edit a doc locally.**
 
 ## 1. Set up your local environment [#set-local]
-
 Running the site locally makes testing and previewing large changes much easier. Here's how to get setup:
 
-1. Install GitHub Desktop
-2. Sign in to GitHub Desktop.
+1. Install GitHub Desktop.
+2. Sign in to GitHub Desktop using your GitHub account.
     - On Macs, click on **GitHub Desktop** in the top left corner of your screen and select **Preferences**. Select the blue **Sign In** button and follow the prompts in the browser window. 
-3. Navigate to the [Docs Site repository](https://github.com/newrelic/docs-website) on GitHub.com. 
-4. Click the green **Code** button and then select **Open with [GitHub Desktop](https://desktop.github.com/)**. 
-5. Choose the location where you want the repo, and this will clone the entire repository to your local machine at the designated path. 
+3. Navigate to the [Docs Site repository](https://github.com/newrelic/docs-website). 
+4. Click the green **Code** button and select **Open with [GitHub Desktop](https://desktop.github.com/)**. 
+5. Choose the location where you want the repo. The default location for this folder is `~/Documents/github`.
+6. Clone the entire repository to your local machine. You will not need to clone the repo again.
```

Comment:
Not quite true. If you deleted the repo off of your local machine, you'd need to clone it again.